### ISSUE TITLE
Not able to make install for ios for architecture arm64

### ISSUE THREAD

**Issue by anuranBarman** [cisco/openh264#3152]
I am trying to build openh264 for ios. I have downloaded the source code from the latest release. then changed the `PREFIX` in makefile to `ios_local`. After that I did `make OS=ios ARCH=arm64` which completed with lot of warning.

After that when I did `sudo make OS=ios ARCH=arm64 install` It exited with error like below
```
clang: warning: libstdc++ is deprecated; move to libc++ with a minimum deployment target of iOS 7 [-Wdeprecated]
ld: library not found for -lstdc++
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make: *** [libopenh264.2.0.0.dylib] Error 1
```

After researching it seems like Apple does not support libstdc++ now, so mac os compatible version is libc++ which is supposed to be installed when you install XCode. My XCode version is `Version 10.2.1 (10E1001)` which is more than enough to get this libc++ I think. What's going wrong here? Please help.

**Comment by huili2** [comment]
you may try this:
 add "-stdlib=libc++" in build/platform-darwin.mk for both CFLAGS and LDFLAGS, something like that:
CFLAGS += -Wall -fPIC -MMD -MP -fstack-protector-all -stdlib=libc++
LDFLAGS += -stdlib=libc++
then see if it is OK.

**Comment by anuranBarman** [comment]
This solves the WARNING in during `make OS=ios ARCH=arm64` but `make OS=ios ARCH=arm64 install` still gives the same error.

**Comment by huili2** [comment]
But the above solution works fine for me... I'm also using XCode 10.2.1. 
Did you do clean for previous build?

**Comment by anuranBarman** [comment]
yes i did `make clean`. Above that I deleted the whole folder and extracted the source code zip again to start from scratch

**Comment by anuranBarman** [comment]
In `Makefile` I see this line `STATIC_LDFLAGS=-lstdc++` .  Should I change this too? It's in line 35

**Comment by huili2** [comment]
It is not necessary to change this flag. So do you still have problem?

**Comment by anuranBarman** [comment]
Yes, I also downloaded the 1.6.0 version. Same problem. From the terminal output its really hard to see which file is actually including it.

**Comment by anuranBarman** [comment]
Aah, finally fixed it by adding the -stdlib=libc++ in the ios make file instead of darwin makefile as you suggested for the LDFLAG.

**Comment by huili2** [comment]
However platform-ios.mk has included darwin at early stage. So still puzzled about it....
Anyway, it is solved. Thanks.

**Comment by anuranBarman** [comment]
Yes I am also thinking about this. I dont know why it happened. Anyway, Thanks for the hint.
